packages/create-llama/templates/components/vectordbs/python/pg/utils.py

```diff
@@ -0,0 +1,23 @@
+import os
+from llama_index.vector_stores import PGVectorStore
+from urllib.parse import urlparse
+from app.engine.constants import PGVECTOR_SCHEMA, PGVECTOR_TABLE
+
+
+def init_pg_vector_store_from_env():
+    original_conn_string = os.environ.get("PG_CONNECTION_STRING")
+    if original_conn_string is None:
+        raise ValueError("PG_CONNECTION_STRING environment variable is not set.")
+
+    # The PGVectorStore requires both two connection strings, one for psycopg2 and one for asyncpg
+    # Update the configured scheme with the psycopg2 and asyncpg schemes
+    original_scheme = urlparse(original_conn_string).scheme + "://"
+    conn_string = original_conn_string.replace(original_scheme, "postgresql+psycopg2://")
+    async_conn_string = original_conn_string.replace(original_scheme, "postgresql+asyncpg://")
```

Comment:
@logan-markewich FYI: I checked the llama-index code - @leehuwuj is correct; a user has to pass both connection strings - so we will keep that code now in `create-llama` - you probably want to move this code here to the `PGVectorStore` constructor, so that a user only has to pass `connection_string`